### ISSUE TITLE
New Release

### ISSUE THREAD

**Issue by dhirschfeld** [jupyterhub/jupyter-rsession-proxy#100]
I'm opening this issue to request a new release of `jupyter-rsession-proxy`.

I was trying to get this working locally with RStudio 1.4 and ran smack into https://github.com/jupyterhub/jupyter-rsession-proxy/issues/93
```
[I 2021-05-31 13:36:32.763 ServerApp] 302 GET /rstudio/ (127.0.0.1) 2431.45ms
[W 2021-05-31 13:36:32.804 ServerApp] 404 GET /auth-sign-in?appUri=%2F (127.0.0.1) 29.28ms referer=http://localhost:8888/lab
```

Following the rabbit-hole, I copy/pasted the code from `master` into my local install and it Just Worked!

I gather that I may run into problems when trying this with JupyterHub but I think having it work locally is enough of a benefit to justify a release now and if the JupyterHub issue is eventually sorted then that could be released separately later.

**Comment by dhirschfeld** [comment]
Ping! Is there any chance someone can release master as-is? The changes since the last release fix a bug I'm running into with RStudio 1.4.

https://github.com/jupyterhub/jupyter-rsession-proxy/compare/v1.2.0...master

**Comment by ryanlovett** [comment]
Thanks for the nudge @dhirschfeld ! I've released master as 1.3.

**Comment by dhirschfeld** [comment]
Thanks for the release! ♥️
